docs/user/managing-fleets.md

```diff
@@ -91,27 +91,29 @@ A fleet's device template contains a device specification that gets applied to a
 
 For example, you could specify in a fleet's device template that all devices in the fleet shall run the OS image `quay.io/flightctl/rhel:9.5`. The Flight Control service would then roll out this specification to all devices in the fleet and the Flight Control agents would update the devices accordingly. The same would apply to the other specification items described in [Managing Devices](managing-devices.md).
 
-However, it would be impractical if *all* of a fleet's devices had to have the *exact same specification*. Flight Control therefore allows templates to contain placeholders that get filled in based on a device's name or label values.
+However, it would be impractical if *all* of a fleet's devices had to have the *exact same specification*. Flight Control therefore allows templates to contain placeholders that get filled in based on a device's name or label values. The syntax for these placeholders matches that of [Go templates](https://pkg.go.dev/text/template), but you may only use simple text or actions (no conditionals or loops, for example). You may reference anything under a device's metadata such as `{{ .Metadata.Labels.key }}` or `{{ .Metadata.Name }}`.
```

Comment:
I it surprising to users that they'd have to use the variable names as in our Go structs instead of those in the YAML files (IOW, they need to use `.Metadata.Name` instead of `.metadata.name`)? in these examples it's just the initial capital letter, but in theory the Go field and YAML field could be named very differently.

Reply:
Good point.  Need to think about how to translate that.